### ISSUE TITLE
Vorta v0.7.0 crashes on Mac OS 10.14.6

### ISSUE THREAD

**Issue by Fackelmann** [borgbase/vorta#641]
**Describe the bug**
After updating Vorta to the newest version (v0.7.0) I cannot run it on my system: MacOS Mojave 10.14.6.

I updated vi a pop up dialogue that notified me there as a newer version available.

**To Reproduce**
1. Open Terminal
2. Execute :
 `> /Applications/Vorta.app/Contents/MacOS/vorta-darwin`

This is the result
```
2020-09-09 20:41:57,942 - vorta.i18n - DEBUG - Loading translation succeeded for ['en', 'en-US', 'en-Latn-US'].
2020-09-09 20:41:57,962 - apscheduler.scheduler - INFO - Scheduler started
2020-09-09 20:41:57,964 - apscheduler.scheduler - INFO - Added job "VortaScheduler.create_backup" to job store "default"
2020-09-09 20:41:57,964 - vorta.scheduler - DEBUG - New job for profile Default was added.
Traceback (most recent call last):
  File "__main__.py", line 44, in <module>
  File "__main__.py", line 37, in main
  File "application.py", line 56, in __init__
  File "application.py", line 90, in open_main_window_action
  File "views/main_window.py", line 50, in __init__
  File "views/schedule_tab.py", line 32, in __init__
  File "views/schedule_tab.py", line 68, in populate_from_profile
  File "views/schedule_tab.py", line 72, in init_wifi
  File "utils.py", line 130, in get_sorted_wifis
  File "network_status/darwin.py", line 48, in get_known_wifis
  File "peewee.py", line 4871, in format_date_time
TypeError: strptime() argument 1 must be str, not datetime.datetime
[718] Failed to execute script __main__
```
**Desktop (please complete the following information):**
- OS: Mac OS Mojave 10.14.6
- Vorta version: 0.7.0
- Borg version: 1.1.13

**Comment by Fackelmann** [comment]
I suspect it might be related to this change:

https://github.com/borgbase/vorta/commit/1b4dea31333057af29bd8703caccdc004b6d202c#diff-aa9fb0be87a4085891df396328fe80bb

**Comment by samuel-w** [comment]
Duplicate of #624, fixed by #625

**Comment by Fackelmann** [comment]
Ops! I should have searched through the closed issues better. Thanks, Samuel.
